File: TestFlutterPlugin/app/src/main/java/top/zcwfeng/testflutterplugin/MyApplication.java
package top.zcwfeng.testflutterplugin;

import android.app.Application;

public class MyApplication extends Application {

    private static MyApplication mApplication = null;


    @Override
    public void onCreate() {
        super.onCreate();
        mApplication = this;




    }

    public static MyApplication getInstance(){
        return mApplication;
    }


}

File: android_kotlin/app/src/main/java/top/zcwfeng/kotlin/MyApplication.kt
package top.zcwfeng.kotlin

import android.app.Application
import androidx.multidex.MultiDex
import androidx.multidex.MultiDexApplication
import com.dbflow5.config.FlowManager

/**
 * 作者：zcw on 2020-01-03
 */
class MyApplication : MultiDexApplication() {
    override fun onCreate() {
        super.onCreate()
        MultiDex.install(this)
        FlowManager.init(this)
    }
}